### DISCUSSION TITLE
微信小程序lite版本，首次播放后，再次play不显示

### DISCUSSION THREAD

**Discussion by wuhongwei1** [Tencent/libpag#2294] [Q&A]
微信小程序lite版本，首次播放后，再次play不显示。
监听到结束后，调用setProgress(0) 然后调用flush()
画布不显示首帧内容，再次调用play()，也不生效

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

如果你还在使用 libpag 且问题仍然存在，麻烦回复一下当前版本号（4.4 / 4.5 系列）和最小复现 demo，我们会优先跟进；如果不再需要跟进，本条就先关闭了。（stale-close）
